c/build_all/linux/build_proton.sh

```diff
@@ -69,8 +69,13 @@ sync_proton ()
     fi
 
     rm $build_root -r -f
-    mkdir $build_root
-    git clone -b $proton_branch $proton_repo $build_root
+    cd /tmp
+    curl -L $proton_repo/archive/$proton_branch.zip -o proton.zip
+    mkdir proton
+    unzip proton.zip -d proton
```

Comment:
I explained my reasoning for this proposed change: it still handles the same general case, but does it faster with less overhead. Obviously it's up to you to ignore it.